src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -112,15 +112,9 @@ public void scrollToSpecificElement(WebElement element) {
   }
 
   public void scrollToElement(WebElement element, int offset) {
-    try {
-      js.executeScript(
-          "var x = $(arguments[0]);" + "window.scroll(0,parseInt(x.offset().top - arguments[1]));",
-          element, offset);
-    } catch (WebDriverException e) {
-      if (e.getMessage().contains(XSSContent.NO_JQUERY_ERROR)) {
-        PageObjectLogging.log("JSError", "JQuery is not defined", false);
-      }
-    }
+    js.executeScript(
+        "window.scroll(0, " + (element.getLocation().getY() - offset) + ");"
```

Comment:
instead of breaking down strings you can use arguments so it would look like this:
```java
int elementPosition = element.getLocation().getY - offset;
js.executeScript("window.scroll(0,arguments[0])",  elementPosition);
```